appveyor.yml

```diff
@@ -42,6 +44,11 @@ test_script:
   - cmd: cd Snowflake.Data.Tests
   # test against .net core 2.0 
   - cmd: ..\OpenCover.4.6.519\tools\OpenCover.Console.exe -target:"dotnet.exe" -returntargetcode -targetargs:"test -f %test_framework% -v n" -register:user -filter:"+[Snowflake.Data]*" -output:"%test_framework%_coverage.xml" -oldStyle
+  # set whitesource api key
+  - cmd: export WHITESOURCE_API_KEY=%wss_secret%
+  # perform whitesource scan
+  - cmd: chmod 755 wss.sh
+  - cmd: ./wss.sh
```

Comment:
@SF-nsfard , this script runs on windows, so I don't think this would work. can you transform it to the batch file?